client/src/components/products/AddProduct.js

```diff
@@ -334,25 +374,33 @@ export default class AddProduct extends Component {
           <FormControl
             required
             component="fieldset"
-            className={classes.formControl}>
+
```

Comment:
Can we remove all empty lines where it has a meaning?

Reply:
cleaned it and added notes for location and maps